src/net.cpp

```diff
@@ -2531,6 +2521,50 @@ void CNode::EndMessage(const char* pszCommand) UNLOCK_FUNCTION(cs_vSend)
     LEAVE_CRITICAL_SECTION(cs_vSend);
 }
 
+bool CConnman::ForEachNode(std::function<bool(CNode* pnode)> func)
```

Comment:
Ack. I meant to document these, but self-doc would be better.

The intention is to use these internally later, for example to replace the
FindNode's which iterate and early-return when possible, to avoid holding
the lock longer than necessary.

But since those changes aren't part of this pull, I think that simply
dropping the bool return and reintroducing it as ForEachNodeContinueIf
later is the way to go.